examples/tools/makefile/makedeps/build_project_makefile.rst

```diff
@@ -0,0 +1,156 @@
+.. _examples_tools_makefile_makedeps_build_project_makefile:
+
+Build a project using Makefile with Conan MakeDeps
+==================================================
+
+In this example, we are going to create a simple logger application that uses one of the most popular C++ libraries:
+`spdlog <https://github.com/gabime/spdlog/>`_  and `fmt <https://fmt.dev/latest/index.html/>`_.
+
+We'll use `Make <https://www.gnu.org/software/make/>`_ as build system, so you should get it installed
+on Linux and Mac before going forward with this example.
+
+Please, first, clone the sources to recreate this project, you can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ on GitHub:
+
+.. code-block:: shell
+
+    git clone https://github.com/conan-io/examples2.git
+    cd examples2/examples/tools/makefile/makedeps/logger
+
+We start with a very simple C++ language project with the following structure:
+
+.. code-block:: text
+
+    .
+    ├── Makefile
+    ├── conanfile.txt
+    └── src
+        └── main.cpp
+
+This project contains a basic `Makefile <https://www.gnu.org/software/make/manual/make.html#Introduction>`_ including the **spdlog** variable dependency and the
+source code for the logger program in *main.cpp*.
+
+Let's have a look at the *main.cpp* file, it only prints a quote said by Corin, but uses ``spdlog::info`` method for it.
+
+.. code-block:: cpp
+    :caption: **main.cpp**
+
+    #include <cstdlib>
+    #include "spdlog/spdlog.h"
+
+
+    int main(void) {
+        spdlog::info("To be a Cimmerian warrior, you must have both cunning and balance as well as speed and strength.");
+        return EXIT_SUCCESS;
+    }
+
+The *Makefile* specifies that ``logger`` is the expected executable and that it should be linked to the ``spdlog`` and ``fmt`` libraries.
```

Comment:
Maybe this comes from copy&paste from other example? I can't see anything about fmt or spdlog there

Reply:
It's a copy and paste indeed, but spdlog and fmt is correct. The spdlog depends on fmt:


```
% conan graph build-order --requires=spdlog/1.12.0

======== Computing dependency graph ========
Graph root
    cli
Requirements
    fmt/10.1.0#51d8d1db43672712db81b146a85ca73d - Cache
    spdlog/1.12.0#248c215bc5f0718402fbf1de126ef847 - Cache
```

Reply:
What I mean is that there's nothing in the Makefile about fmt or spdlog, it's a bit confusing for me

Reply:
@czoido I just updated the statement for global variables. Then, mentioned about separated variables.